core/new-gui/src/app/dashboard/component/feature-container/saved-workflow-section/ngbd-modal-workflow-executions/ngbd-modal-workflow-executions.component.html

```diff
@@ -80,11 +80,35 @@ <h4 class="modal-title">Execution results of Workflow {{workflowName}}</h4>
             nzTooltipPlacement="top"
```

Comment:
try this line next to `nz-table` `<nz-table [nzScroll]="{ x: '800px' }"` to add the horizontal scrolling, but please adjust the width of the columns to be small for those of only containing icons e.g., status, checkbox, delete, favourite columns to be smaller and the other columns to be wider.